### PR TITLE
 Fix failed functional B2B tests

### PR DESCRIPTION
**Pull request by engcom-Golf**
### Related Pull Requests
https://github.com/magento/partners-magento2ee/pull/127

### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by engcom-Golf** [comment]
@magento run all tests

**Comment by engcom-Golf** [comment]
@magento run all tests

File: app/code/Magento/Catalog/Test/Mftf/Test/AdminMoveCategoryAndCheckUrlRewritesTest.xml [hunk 1]
```diff
@@ -20,6 +20,7 @@
         <before>
             <actionGroup ref="LoginAsAdmin" stepKey="loginToAdminPanel"/>
             <createData entity="_defaultCategory" stepKey="createDefaultCategory"/>
+            <magentoCLI command="cron:run --group=index" stepKey="runIndexerCron"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Test/Mftf/Test/AdminMoveCategoryAndCheckUrlRewritesTest.xml]: https://github.com/magento/magento2/pull/26395/files#diff-173e5d33719194f591cb7818fb12ee85R32

**Comment by lbajsarowicz** [comment]
You can also include this one: https://github.com/magento/magento2/pull/26407/files to the PR :+1:

**Review by lbajsarowicz — APPROVED** [review]
Great enhancement to:
https://github.com/magento/magento2/pull/26407
https://github.com/magento/magento2/pull/26395

Kudos for using `cron:run --group=index` instead of `index:reindex && cache:flush` :+1:

**Comment by engcom-Alfa** [comment]
QA not applicable

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6634](https://jira.corp.magento.com/browse/ENGCOM-6634) has been created to process this Pull Request

**Comment by engcom-Golf** [comment]
I closing this PR as no actual any more, test passed on 2.4-develop branch

**Comment by lbajsarowicz** [comment]
@engcom-Golf  In my opinion - even if tests are (accidentally or not) passing, these changes are important for improving Tests stability.

**Pull request closed without merging**